### DISCUSSION TITLE
Interesting contracts argument?

### DISCUSSION THREAD

**Discussion by ndtrung00** [fuzzland/ityfuzz#384] [Q&A] [answered]
Does it matter how we sort the contracts in that arguments? 

-> i see sometimes in your docs, for example: the fuzzing command for ARA: ` ityfuzz evm -o -t 0x55d398326f99059fF775485246999027B3197955,0x7BA5dd9Bb357aFa2231446198c75baC17CEfCda9,0x13f4EA83D0bd40E75C8222255bc855a974568Dd4,0x5542958FA9bD89C96cB86D1A6Cb7a3e644a3d46e,0x98e241bd3be918e0d927af81b430be00d86b04f9,0x7ba5dd9bb357afa2231446198c75bac17cefcda9 -c bsc --onchain-block-number 29214010 -f -i -p --onchain-etherscan-api-key FJMFRFSI3Z8WP9YVU6IBRVZ35IQKB5EPTH`

the vulnerable contract comes twice: `0x7ba5dd9bb357afa2231446198c75bac17cefcda9`

is that on purpose? because i cant see how or whether ityfuzz differentiates which one is attack contract, which one is vulnerable contract.

**Comment by 0xAWM** [comment, marked as answer]
The order of addresses makes no difference
